conan/tools/build/flags.py

```diff
@@ -469,3 +482,33 @@ def _cppstd_intel_cc(_, cppstd):
             "20": v20, "gnu20": vgnu20,
             "23": v23, "gnu23": vgnu23}.get(cppstd)
     return f'-std={flag}' if flag else None
+
+
+def to_cppstd_flag(compiler, compiler_version, cppstd):
```

Comment:
Moved from helper.py. No changes at all.

Reply:
But the problem is that this can be confusing here, because these flags are exclusively in Meson format, but this looks general thing. It should be in the ``tools.meson`` scope, better than here maybe?

Reply:
True enough, but then I'll rename it as `to_meson_cpp_flag` to be clear.

Reply:
I just reverted that commit, it will keep that helper in meson folder.